pkg/minishift/addon/manager/addon_manager_test.go

```diff
@@ -114,7 +115,8 @@ func TestAddVarDefaultsToContext(t *testing.T) {
 
 	testAddonMap := getTestAddonMap("test", "test description", expectedVarName, varDefault, "")
 
-	addOnMeta := getAddOnMetadata(testAddonMap, t)
+	addOnMeta, err := addon.NewAddOnMeta(testAddonMap)
+	assert.NoError(t, err, "Failed to create new addon meta")
```

Comment:
"Failed to create new addon meta" => "Failed to create addon meta" no need to put `new` here `NewAddonMeta` is just a function which returns add-on metadata.

Reply:
I thought it was some kind of a constructor, so went with `new`, anyways will change it.